lib/tether/netfilter/netfilter_linux.go

```diff
@@ -158,7 +160,7 @@ func (r *Rule) args() ([]string, error) {
 	return args, nil
 }
 
-func iptables(ctx context.Context, args []string) error {
+func iptables(ctx context.Context, args []string) (tether.UtilityFn, error) {
 	args = append(args, "-w")
```

Comment:
@willsherwood I think it's a useful addition anyway - it will avoid issues if the user is attempting to update a rule when a Reload happens for example. Absent any compelling reason I will leave it in.